[2023-02-04 07:40:32,716] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T07:40:28.448176+00:00 [queued]>
[2023-02-04 07:40:32,729] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T07:40:28.448176+00:00 [queued]>
[2023-02-04 07:40:32,730] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:32,732] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 07:40:32,732] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:32,750] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2023-02-04 07:40:28.448176+00:00
[2023-02-04 07:40:32,755] {standard_task_runner.py:52} INFO - Started process 6464 to run task
[2023-02-04 07:40:32,759] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2023-02-04T07:40:28.448176+00:00', '--job-id', '163', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag_test.py', '--cfg-path', '/tmp/tmpmqjrnl20', '--error-file', '/tmp/tmpsgr611x4']
[2023-02-04 07:40:32,759] {standard_task_runner.py:80} INFO - Job 163: Subtask insert_data
[2023-02-04 07:40:32,837] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T07:40:28.448176+00:00 [running]> on host 39df186e1726
[2023-02-04 07:40:32,919] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T07:40:28.448176+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-04T07:40:28.448176+00:00
[2023-02-04 07:40:32,931] {logging_mixin.py:115} INFO - rate_date:  2023-02-04
[2023-02-04 07:40:32,932] {logging_mixin.py:115} INFO - value_:  23510.60869599999932
[2023-02-04 07:40:32,941] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-02-04 07:40:32,964] {python.py:173} INFO - Done. Returned value was: None
[2023-02-04 07:40:32,978] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20230204T074028, start_date=20230204T074032, end_date=20230204T074032
[2023-02-04 07:40:33,011] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 07:40:33,064] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
